pkg/apis/core/validation/validation.go

```diff
@@ -2337,6 +2358,11 @@ func validateContainerResourceDivisor(rName string, divisor resource.Quantity, f
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("divisor"), rName, "only divisor's values 1, 1k, 1M, 1G, 1T, 1P, 1E, 1Ki, 1Mi, 1Gi, 1Ti, 1Pi, 1Ei are supported with the local ephemeral storage resource"))
 		}
 	}
+	if strings.HasPrefix(rName, "requests.hugepages-") || strings.HasPrefix(rName, "limits.hugepages-") {
```

Comment:
are these prefixes constants or can this prefix check be made a "is hugepages resource" method